src/style.css

```diff
@@ -1,3 +1,90 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+ol,
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
   margin: 0;
+  list-style: none;
+}
```

Comment:
You should not use tag names for styling. It's better to use class selectors. This is a code quality issue because using tag names can lead to unintended styling consequences if new elements are added to the HTML. Instead, assign classes to elements that need specific styling. This will make your CSS more maintainable and scalable.